pandas/core/series.py

```diff
@@ -87,6 +87,7 @@
 from pandas.core.dtypes.dtypes import (
     CategoricalDtype,
     ExtensionDtype,
+    SparseDtype,
```

Comment:
Surprising to see this here. Maybe more motivation for https://github.com/pandas-dev/pandas/issues/56518

Reply:
yes, a bit unfortunate. Alternatively, the sparse issue was previously xfailed and could be xfailed again in order to avoid this:

https://github.com/pandas-dev/pandas/pull/51777/files#diff-16458c6ca46c7c2e895b1fad1300009d8285d19cdbef3023a33f520d15bc7085L333